server/app/api.py

```diff
@@ -1821,7 +1763,8 @@ def get_courses(self, course, user, data):
     def get_students(self, course, user, data):
         query = models.Participant.query(
             models.Participant.course == course.key,
-            models.Participant.role == 'student')
+            models.Participant.role == 'student',
+            models.Participant.status != 'inactive')
```

Comment:
@alvinwan I really hope that there is some way of solving this that doesn't involve you going through it all. If I can do it, please let me know. I guess I can just look in the master repo on GitHub. 
I can se tons of people having the same issue with "Already up-to-date" even when all files aren't the same, but I still haven't found a solution that works. 
If it is just a question of whether you or I find the changes manually, let me know - then I'll do it tomorrow.